app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/insurance/DocumentRow.kt

```diff
@@ -0,0 +1,71 @@
+package com.hedvig.android.core.ui.insurance
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.hedvig.android.core.designsystem.newtheme.SquircleShape
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.icons.Hedvig
+import com.hedvig.android.core.icons.hedvig.small.hedvig.ArrowNorthEast
+
+@Composable
+fun DocumentRow(
+  modifier: Modifier = Modifier,
+  title: String,
+  subTitle: String,
+  onClick: () -> Unit,
+) {
+  Row(
+    verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.SpaceBetween,
+    modifier = modifier
+      .background(
+        shape = SquircleShape,
+        color = MaterialTheme.colorScheme.background
+      )
+      .fillMaxWidth()
+      .padding(horizontal = 16.dp, vertical = 12.dp)
+      .clickable { onClick() },
+  ) {
+    Column {
+      Row {
+        Text(title)
+        Spacer(modifier = Modifier.padding(1.dp))
+        Text("PDF", fontSize = 11.sp)
```

Comment:
For this you can reuse what I had done here https://github.com/HedvigInsurance/android/blob/42a2b7379f8237e42e5c5edf7d67541b401d46c5/app/feature/changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/destination/ChangeAddressOfferDestination.kt#L385-L398
Basically you can make parts of a text feel like a "superscript" by using an annotated string.

Here's a SOE post about this https://stackoverflow.com/a/66801935/9440211. I just didn't use superscript I remember since it was not looking as we wanted it to look. You can see for yourself if it is what we want in this scenario though.